# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libtai/libtai-0.60-0002-errno.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff --git a/leapsecs_read.c b/leapsecs_read.c
index 3919f35..96a4d64 100644
--- a/leapsecs_read.c
+++ b/leapsecs_read.c
@@ -3,7 +3,6 @@
 #include <fcntl.h>
 #include <stdlib.h> /* free, malloc */
 #include <errno.h>
-extern int errno;
 #include "tai.h"
 #include "leapsecs.h"
 
